website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
Or today ... :rabbit: 

Why does [`OsfWebRenderer__index`](https://github.com/CenterForOpenScience/osf.io/blob/cef7a1f89871bf624473b556ff70157b6701a1bf/website/util/__init__.py#L125) point to the index view function?